components/Results/results.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable @next/next/no-img-element */
+import { useState } from "react"
```

Comment:
This file is just a bunch of design stuff. Main thing is using the new provider to get token id and image uri